src/Reflection/Node.php

```diff
@@ -77,11 +86,17 @@ public function getParent(): ?self
         return $this->parent;
     }
 
+    /**
+     * @return mixed
+     */
     public function getValue()
     {
         return $this->value;
     }
 
+    /**
+     * @param mixed $value
+     */
     public function setValue($value): void
```

Comment:
Docblock says `return mixed`, but declaration says `void`...

Reply:
The argument is hinted with `mixed`. Return type hint in DocBlock is obsolete due to return type declaration.